tools/shared.py

```diff
@@ -169,6 +169,13 @@ def check_returncode(self):
 
 
 def run_process(cmd, check=True, input=None, *args, **kw):
+  """Runs a subpocess returning the exit code.
+
+  By default this function will raise an exception on failure.  Therefor this should only be
+  used if you want to cratch such failures.  For most subprocess failures should fatal and you
```

Comment:
typo in `catch`

`For most subprocess failures should fatal` => `Most subprocess failures should be`, I think?